romana-install/roles/romana/install-master/templates/config/romana.conf.yml

```diff
@@ -63,6 +63,9 @@ services:
     config:
       lease_file : "/etc/ethers"
       wait_for_iface_try: 6
+      store:
+        type: sqlite3
+        database: /var/agent.db
```

Comment:
BTW, It really should be in-memory as it was before https://github.com/paninetworks/core/commit/dc38dee56285e1ccbc100bd7bd47ddc849539f22#diff-56677df749cde51512d399095189b5b6) , but somehow it stopped working and I reverted here: https://github.com/paninetworks/core/commit/dbc7c6eda631eeb4e37714b4c9ae9c63d3fb5daf#diff-56677df749cde51512d399095189b5b6

Doesn't matter for purposes of this PR...